src/cunumeric/index/advanced_indexing.cc

```diff
@@ -79,11 +79,6 @@ struct AdvancedIndexingImplBody<VariantKind::CPU, CODE, DIM, OUT_TYPE> {
       if (index[p] == true) { size++; }
     }
 
-    if (0 == size) {
```

Comment:
I remove this block since we our output region should be specific shape even if its size is 0